docs/documentation/using-notify.md

```diff
@@ -0,0 +1,48 @@
+To set it locally, run this command in your Terminal, in the root
+folder of your prototype:
+```shell
+   echo NOTIFYAPIKEY=xxxxxxx >> .env
+````   
+(where xxxxxxx is a key you’ve created in Notify). This creates a file 
+named `.env` which contains your API key.
+
+To set it on Heroku, go to the settings page on your app, click
```

Comment:
Is it clear that users will need to do this for Heroku (if they're using Heroku) _as well as_ locally, because their .env file won't (shouldn't) be committed? At the minute it could be interpreted as an either/or thing.